app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1>Categories</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Icon</th>
+      <th>Total Amount</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @categories.each do |category| %>
+      <tr>
+        <td><%= category.name %></td>
+        <td><%= category.icon %></td>
+        <td><%= category.expenses.sum(:amount) %></td>
+        <td><%= link_to 'Show', category %></td>
+        <td><%= link_to 'Edit', edit_category_path(category) %></td>
+        <td><%= link_to 'Destroy', category, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+
```

Comment:
- [ ] Kindly improve your new category page to match the form design.

  ![image](https://github.com/Tracy-miranja/budget/assets/36542986/afe59ea3-0cf7-4b52-aa2d-27f3a26d53d6)